proposals/templates/proposals/vue_templates/proposal_list.html

```diff
@@ -64,24 +64,17 @@ <h4>
                          title="{% trans 'Verwijderen' %}">
                 </a>
             </template>
-
+<template
+                v-if="proposal.status >= context.proposal.SUBMITTED_TO_SUPERVISOR"
+>
             <a
                 :href="$url('proposals:pdf', [proposal.pk])"
                 target="_blank"
-                v-if="proposal.pdf"
             >
                 <img src="{{ img_pdf }}"
                      title="{% trans 'Inzien' %}">
             </a>
-            <a
-                v-else
-                :href="$url('proposals:pdf', [proposal.pk])"
-                target="_blank"
-            >
-                <img src="{{ img_pdf }}"
-                         title="{% trans 'Inzien' %}">
-            </a>
-
+</template>
```

Comment:
So two things:
1. Please indent this properly, or:
2. Remove this, as you don't need this template tag here. You can directly apply this v-if to the <a> element. 

The <template> example above is for clarity, as there are 2 links with the same condition. <template> is a no-op Vue tag that allows you to group them in your template, without adding something in the output. (Which is needed here for CSS reasons)

Reply:
Noted, thanks. Removed the template tag